### PR TITLE
Switched Basic Project information page and privacy and security to b3

### PR DESCRIPTION
**Pull request by kkrampa**
@biyeun 

Project information page:

Privacy and security:

Exchange:

### CODE REVIEW

File: corehq/apps/domain/forms.py [hunk 1]
```diff
@@ -665,6 +683,26 @@ class PrivacySecurityForm(forms.Form):
         help_text=ugettext_lazy("Allow unknown users to request web access to the domain."),
     )
 
+    def __init__(self, *args, **kwargs):
+        super(PrivacySecurityForm, self).__init__(*args, **kwargs)
+        self.helper = FormHelper(self)
+        self.helper.form_class = 'form-horizontal'
+        self.helper.label_class = 'col-sm-3 col-md-2'
+        self.helper.field_class = 'col-sm-9 col-md-8 col-lg-6'
+        self.helper[0] = twbscrispy.PrependedText('restrict_superusers', '')
```
Comment (kkrampa) [inline corehq/apps/domain/forms.py]: @biyeun Without this hack boolean field is rendered in that way: 



I'm not sure maybe I'm doing something wrong but I've checked existing bootstrap3 form in hq and in my browser it also looks bad: (e.g. https://www.commcarehq.org/a/lukasz-test/indicators/import/). Maybe it's related to this problem: https://github.com/maraujop/django-crispy-forms/issues/225?

File: corehq/apps/domain/forms.py [hunk 2]
```diff
@@ -26,7 +26,9 @@
 from crispy_forms.bootstrap import FormActions, StrictButton
 from crispy_forms.helper import FormHelper
 from crispy_forms import layout as crispy
+from crispy_forms import bootstrap as twbscrispy
 from corehq.apps.style import crispy as hqcrispy
+from crispy_forms import bootstrap as twbscrispy
```
Comment (dimagimon) [inline corehq/apps/domain/forms.py]: F811 redefinition of unused 'twbscrispy' from line 29

**Comment by kkrampa** [comment]
Added exchange page since they are all extends same template

**Comment by biyeun** [comment]
@kkrampa posting the basic settings form doesn't appear to have any effect

**Comment by biyeun** [comment]
Privacy and Security is functional and looks good :+1:

**Comment by biyeun** [comment]
Don't forget that any buttons with _just_ the class `btn`, like the cancel button, needs to now have `btn-default` added to it in B3. (so `btn` -> `btn btn-default`) Thanks!

**Comment by kkrampa** [comment]
@biyeun Problem with posting form was connected with fact that bootstrap_fieldset tag was rendering only visible fields (https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/templates/hqstyle/forms/basic_fieldset.html#L4) whereas crispy is rendering all fields. Fixed this in https://github.com/lwyszomi/commcare-hq/commit/3d7faca1802dbfabb60c80955972171c3eec901e

**Comment by biyeun** [comment]
Great, tested, looks good :+1:

**Pull request merged**